test/Sentry.EntityFramework.Tests/Sentry.EntityFramework.Tests.csproj

```diff
@@ -5,12 +5,12 @@
   </PropertyGroup>
 
   <!-- Running these tests on Mono fail -->
-  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+  <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('windows'))">
```

Comment:
Here's another case where we were checking incorrectly.  If `OS` wasn't defined, then the tests din't run for this target at all on any OS.